.github/workflows/builder_docker-based_slsa3.yml

```diff
@@ -79,8 +79,12 @@ on:
       build-outputs-name:
```

Comment:
Thanks @asraa. Do you want to rename `build-outputs-name` to `build-artifacts-name` too?

Both work for me.

Reply:
Follow-up in https://github.com/slsa-framework/slsa-github-generator/issues/1735 !